scripts/validation/src/parser/pair_parser.py

```diff
@@ -14,9 +15,15 @@ def __init__(self, pairs_fname: str, image_dir: str) -> None:
 
     def compute_pairs(self) -> Iterable[Pair]:
         with open(self.pairs_fname, 'r', encoding='utf-8') as f:
-            next(f)  # skip first line, which contains metadata
+            next(f) # pylint: disable=stop-iteration-return
+                    # skip first line, which contains metadata
             for line in f:
-                yield self._compute_pair(line)
+                try:
+                    pair = self._compute_pair(line)
+                except FileNotFoundError as e:
+                    logging.exception(e)
```

Comment:
`logging.exception('your-message-here')` will automatically also log the exception that's currently raised, so no need to assign it to a variable here.